### ISSUE TITLE
Image:  missing `crossOrigin`, `referrerPolicy`, `loading` and `useMap` 

### ISSUE THREAD

**Issue by manuelpoelzl** [primefaces/primereact#4273]
### Describe the feature you would like to see added

It would be great if the Image component would support the crossOrigin attribute of the native image component.
Because currently it is not possible to use the primereact image component if `use-crendtials`  is required.

### Is your feature request related to a problem?

Not directly related to a problem, but for example, if you retrieve your images from a server and you want them to have a preview functionality, this cant be accomplished now, since the native `<img/>` tag with the `crossOrigin` attribute has to be used.

### Describe the solution you'd like

Basically the same options as on the native image tag.

### Describe alternatives you have considered

Not using the primereact image component and falling back to the native one.
Or changing the cors options.

### Additional context

Sorry if this option does already exists, wasn't able to find it in the [docs](https://primereact.org/image/).

**Comment by melloware** [comment]
Adding `crossOrigin`, `referrerPolicy`, `loading` and `useMap` props were all missing from core Image.

**Comment by manuelpoelzl** [comment]
Thanks for the quick implementation @melloware 😄

**Comment by manuelpoelzl** [comment]
I have just tried the changes, and in the preview it works without problems now, only the real opening of the image still shows an incorrect image icon:

Preview:
![image](https://user-images.githubusercontent.com/46539758/234993893-5be65490-2b31-439a-b0dd-100172a4b818.png)

Opened image:
![image](https://user-images.githubusercontent.com/46539758/234993957-76cf2456-3e8b-4da9-a64c-9d780bbdf92d.png)


I also tried to specify the zoomSrc, unfortunately without success.
Should I open a new issue for this or could something be wrong here on my side?

**Comment by melloware** [comment]
New issue please.

**Comment by melloware** [comment]
Yep I see the issue. The props are not being applied to the big image only the preview
